### ISSUE TITLE
Resolve WARNING on npm install in Quickstart

### ISSUE THREAD

**Issue by ALRubinger** [TBD54566975/developer.tbd.website#418]
Dependencies: 

* https://github.com/TBD54566975/web5-js/issues/45
* #409 

Based on their conclusion to this, update docs where you're putting the notice to users about this warning on `npm install`

**Comment by ALRubinger** [comment]
@bobbilee19 The dependency above has been resolved by @frankhinek and @csuwildcat - thanks, y'all!

After upgrading the Quickstart to `0.6.0`, expected behavior is that the security vulnerability warnings should no longer be present. To close this issue, all we gotta do is confirm that's the case, and update the Quickstart to remove the warning.

**Comment by ALRubinger** [comment]
Updated issue description to reflect the dependency on #409 - we can't remove the warning until we upgrade Docs/Quickstart to use Web5 `0.6.0`, which contains the fix for the underlying problem we're `WARN`ing about.

**Comment by ALRubinger** [comment]
Verified that `npm install` using `0.6.0` indeed removes the security vulnerability:

```
npm install @tbd54566975/web5@0.6.0
npm WARN deprecated multibase@4.0.6: This module has been superseded by the multiformats module

added 173 packages, and audited 174 packages in 22s

18 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

**Comment by dayhaysoos** [comment]
This will be addressed by #398

**Comment by ALRubinger** [comment]
Done via #398
